src/gui/captureHandler/MultiCaptureHandler.cpp

```diff
@@ -300,6 +311,17 @@ void MultiCaptureHandler::saveTab(int index)
 	saveAt(index, true);
 }
 
+void MultiCaptureHandler::saveAllTabs()
```

Comment:
Can you add a test for this? There should be already a test class for SingleCaptureHandler and MultiCaptureHandler. If it's to much work I can do it too.